sonar-text-dotnet/src/SonarLint.Secrets.DotNet/Rules/Matching/analysis_begin_step.ps1

```diff
@@ -0,0 +1,25 @@
+Set-Location $env:PROJECT_DIR
```

Comment:
This file seems to be now in 2 places. Is this intended?

Reply:
Glitch in the matrix.
Fixed!